README.md

```diff
@@ -66,13 +69,87 @@ findPorts({ firefoxOSSimulator: true }).then(function(results) {
 }, function(err) {
   console.log(err);
 });
+```
+
+The output from the above code might look like the following:
+```javascript
+[ { type: 'b2g', port: 56567, pid: 45876 },
+  { type: 'firefox', port: 6000, pid: 3718 },
+  { type: 'device', port: 8001, deviceId: '3739ced5' } ]
+```
 
+Including the `detailed` option interrogates each runtime for additional information:
+```javascript
 // Returns only Firefox OS simulators, with extra detailed output
-findPorts({ firefoxOSSimulator: true, detailed: true }).then(function(results) {
+findPorts({
+  firefoxOSSimulator: true, 
+  firefoxOSDevice: true,
+  detailed: true
+}).then(function(results) {
   console.log(results);
 });
 ```
 
+The output from this code might look like so:
```

Comment:
Again, a bit more flourish than needed in docs.

"Detailed output includes a **lot** more info:"